node-operators/block-producers/operational-tasks/upgrade-your-node/upgrade-docker.md

```diff
@@ -0,0 +1,100 @@
+
+=== "Generic"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    moondancelabs/tanssi \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+
+=== "Intel Skylake"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    --entrypoint "/tanssi/tanssi-node-skylake" \
+    moondancelabs/tanssi \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+=== "AMD Zen3"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    --entrypoint "/tanssi/tanssi-node-znver3" \
+    moondancelabs/tanssi \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+
+### Specifying a Version Tag {: #specifying-a-version-tag }
+
+The Docker commands above will automatically fetch the latest release version tag, but if you wanted to specify a [particular version tag](https://hub.docker.com/r/moondancelabs/tanssi/tags){target=\_blank}, you can do so by appending the version tag to the image name. For example, if you wanted to fetch version `v0.5`, rather than specifying the image name as `moondancelabs/tanssi`, you would indicate `moondancelabs/tanssi:v0.5` as the image name.
+
+The complete commands with specific version tags are thus as follows:
+
+=== "Generic"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    moondancelabs/tanssi:v0.5 \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+
+=== "Intel Skylake"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    --entrypoint "/tanssi/tanssi-node-skylake" \
+    moondancelabs/tanssi:v0.5 \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+=== "AMD Zen3"
+
+    ```bash
+    docker run --network="host" -v "/var/lib/dancebox:/data" \
+    -u $(id -u ${USER}):$(id -g ${USER}) \
+    --entrypoint "/tanssi/tanssi-node-znver3" \
+    moondancelabs/tanssi:v0.5 \
+    --8<-- 'text/node-operators/block-producers/onboarding/run-a-block-producer/docker-command.md'
+    ```
+
+## Conclusion {: #conclusion }
+
+And that's it! You've successfully upgraded your Tanssi Node. You can subscribe to email notifications of new releases by clicking **Watch** on the [Tanssi GitHub Repo](https://github.com/moondance-labs/tanssi){target=\_blank} and selecting **Custom** notifications and checking the box for **Releases**. 
```

Comment:
```suggestion
And that's it! You've successfully upgraded your Tanssi node.

## Subscribe to Release Notifications {: #subscribe }

You can subscribe to email notifications of new releases by clicking **Watch** on the [Tanssi GitHub Repo](https://github.com/moondance-labs/tanssi){target=\_blank} and selecting **Custom** notifications and checking the box for **Releases**. 
```

I'm not a big fan of the conclusion section, because we don't typically have them. I'd prefer something like this

Reply:
If you think this is alright, please update the link to this section in the intro